### DISCUSSION TITLE
How to restart a database paused due to inactivity?

### DISCUSSION THREAD

**Discussion by dgknca** [supabase/supabase#12686] [Questions] [answered]
After a 2 week break, I started working on my supabase project again and I saw that the signup API calls returns 500. I think my database was paused because I'm on the free tier. But I can't figure out how to restart it. There is no guide anywhere. How can I restart my project?

**Comment by GaryAustin1** [comment]
![image](https://user-images.githubusercontent.com/54564956/221419922-c12ec09f-f606-403e-90c0-0110e30f2af2.png)
You should see this on your main projects screen and if you click it you will be given option to restore.

**Comment by GaryAustin1** [comment, marked as answer]
It is possible you are not paused and getting 500 errors on signup because of an auth.users trigger function.  If you can get to your project's dashboard and see the logs tab, then you are likely not paused.  In that case look in the Postgres log after the error for information on why your signUp is getting an error.

**Reply by dgknca** [reply]
yes, you're right. it's not paused. When I looked at the Postgres logs, I saw that the error was caused by something else. Thank you so much !
